src/main.ts

```diff
@@ -16,6 +16,7 @@ async function bootstrap() {
     logger: process.env.DEBUG ? ['error', 'warn', 'log', 'verbose', 'debug'] : ['error', 'warn', 'log'],
   });
 
+  process.on('uncaughtException', (error) => console.error(error));
```

Comment:
This looks dangerous. I suggest:
```suggestion
  process.on('uncaughtException', (error) => {
    console.error(error);
    process.exit(1);
  });
```